### ISSUE TITLE
Throwing and Munitions Overhaul

### ISSUE THREAD

**Issue by bombasticSlacks** [bombasticSlacks/Aspirant#124]
Thrown weapons differentiate between harming / utility. 

How do you calculate thrown damage bonus?

Sling should allow using accuracy.

```
Sling trait proposal: go all in on ranged weapon + throw skill modifier
Loading 0
Capacity 1 (Throwable munition)
Does subtract basic bonus to damage
Uses basic/artisan bonus from munition (??? need to ruminate on quality of munitions)
You can Substitute accuracy for athletics when throwing with this weapon.
Throwing skill checks using this weapon gain +1 range.

So, with basic scaveneged rocks (basic throwable munition): damage bonus = (2 x 3) -1 = 5 = damage 1 (-1)
with artisan sling bullets, damage bonus = (2 x 3)+1 = 7 = damage 2 (-5)

When throwing my bombs, I just do an accuracy check to throw up to close range to do the damage listed by the ritual (res +2 = 6 = 2 (-5)) 
so basically my bomb ritual (without modification) is basically just crafting sparkling artisan sling stones
the middle bullet point is so that a basic sling has -1 penalty to damage for being shoddy
```

**Comment by bombasticSlacks** [comment]
Other thoughts:

Simplify as:
Substitute accuracy.
Substitute strength for sling size.
Increase range by 1 increment.

**Comment by bombasticSlacks** [comment]
Also mentions of basic vs martial munitions.

**Comment by akrieger** [comment]
More ideas:

Maybe munitions don't have a grade like basic/artisan, they just have a flat effect. So eg.
'scavenged rocks/wooden arrows: throw for -1 damage bonus'
'lead sling bullets/metal tipped arrows: throw/shoot for +1 damage bonus'
'broadhead arrows: shoot for +1 damage bonus, rending'
'explosive pellet: throw for <fixed damage> rending'

Keep it at 2*size+MIN(strength, size) damage bonus.
Range increment +1 for size 1-2, +2 for size 3-4?
